openrpc.yaml

```diff
@@ -97,6 +97,7 @@ methods:
                 - 'https://rpc.ankr.com/gnosis'
               iconUrls:
                 - 'https://xdaichain.com/fake/example/url/xdai.svg'
+                - 'https://xdaichain.com/fake/example/url/xdai.png'
```

Comment:
Is there a reason why we include two iconUrls here?

Reply:
Not sure. I just merged the duplicate wallet_addEthereumChain entries and this second iconUrl was in one of them